README.md

```diff
@@ -19,5 +19,49 @@
 6. Edit the newly created CSharp script, and add `using TestFairyUnity;` to the import section, and a call to `TestFairy.begin()` with your app token. You can find your app token in  [Account Settings](https://app.testfairy.com/settings/#apptoken) page.
 
   ![Step 4](/Images/step4.png)
+
+```
+using UnityEngine;
+using System.Collections;
+using TestFairyUnity;
+
+public class mainCameraScript : MonoBehaviour {
+
+    // Use this for initialization
+    void Start () {
+        TestFairy.begin("0ddd54741fc830787fb8e1a8232a49733ce9759b");
+    }
+
+    ...
+}
+```
   
-Save, build and run.
+7. At minimum, TestFairy requires the `INTERNET` permission for your Android build. You can copy a version of your AndroidManifest.xml from `<root>/Temp/StagingArea/AndroidManifest.xml` into `<root>/Assets/Plugin/Android` directory. From here, edit `AndroidManifest.xml` with the following line
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+Additional features may require extra persmissions given below
+
+```xml
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+<uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
+<uses-permission android:name="android.permission.BATTERY_STATS"/>
+<uses-permission android:name="android.permission.GET_ACCOUNTS"/>
+<uses-permission android:name="android.permission.RECORD_AUDIO"/>
+<uses-permission android:name="android.permission.READ_PHONE_STATE"/>
+<uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES"/>
+
+<uses-feature android:name="android.hardware.camera"/>
```

Comment:
We don't use camera or opengl es 2.0. In the future we will re-do the front camera and audio code, and it will be documented per-feature. Not something people drag and drop to have unity enabled.